docs/AboutUs.md

```diff
@@ -5,19 +5,18 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
-
 ## Project team
 
-### John Doe
+### Ivan Wong
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/ivanwong.png" width="200px">
```

Comment:
Should the file name of your profile picture be `wpinrui.png` instead?

Reply:
Yes! Changes made, thanks.